h3/src/quic.rs

```diff
@@ -122,15 +182,15 @@ pub trait SendStream<B: Buf> {
     fn reset(&mut self, reset_code: u64);
 
     /// Get QUIC send stream id
-    fn id(&self) -> StreamId;
+    fn send_id(&self) -> StreamId;
```

Comment:
Avoids disambiguity with `id` on `RecvStream` as a send stream id if required for the webtransport control stream to identify the session